src/modules/mim/osconfigconfiguration.json

```diff
@@ -0,0 +1,105 @@
+{
+  "name": "OsConfigConfigurationModel",
+  "type": "MimModel",
+  "contents": [
+    {
+      "name": "OsConfigConfiguration",
+      "type": "mimComponent",
+      "contents": [
+        {
+          "name": "desiredConfiguration",
+          "type": "mimObject",
+          "desired": false,
+          "schema": {
+            "fields": [
+              {
+                "name": "refreshInterval",
+                "schema": "integer"
+              },
+              {
+                "name": "localManagementEnabled",
+                "schema": "boolean"
+              },
+              {
+                "name": "fullLoggingEnabled",
+                "schema": "boolean"
+              },
+              {
+                "name": "commandLoggingEnabled",
+                "schema": "boolean"
+              },
+              {
+                "name": "iotHubProtocol",
+                "schema": {
+                  "type": "enum",
+                  "valueSchema": "integer",
+                  "enumValues": [
+                    { 
+                      "name": "auto",
+                      "enumValue": 0
+                    },
+                    {
+                      "name": "mqtt",
+                      "enumValue": 1
+                    },
+                    {
+                      "name": "mqttWebSocket",
+                      "enumValue": 2
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        },
+        {
+          "name": "modelVersion",
+          "type": "mimObject",
+          "desired": true,
```

Comment:
Is this right? This is not desired, right?

Reply:
Right